src/GroupSocket.js

```
@@ -231,16 +231,16 @@ export default class GroupSocket {
           }
           break;
         default: { // Free Response
```

probably should handle the `FREE_RESPONSE` state manually here, just in case further question types are added
maybe throw an exception in the `default` case with an `Unimplemented` message?

---

addressed in latest push